waspls/README.md

```diff
@@ -0,0 +1,32 @@
+# Waspls
+
+This directory contains source code of the `wasp` language server (aka `waspls`)
+and this README is aimed at the contributors to the project.
+
+## Overview
+
+`waspls` is implemented in Haskell. It depends on `waspc` for parsing and
+analyzing wasp source code. Cabal is currently configured to look in `../waspc`
+for the `waspc` package, so do not move `waspls` out of the `wasp` repo without
+adjusting this line in `cabal.project`.
+
+## Usage
+
+Use in any place an LSP server can be used. Usage:
+
+```
+Usage: waspls [version] [-v|--version] [--log LOG_FILE] [--stdio]
+
+  LSP Server for the Wasp language
+
+Available options:
+  -h,--help                Show this help text
+  -v,--version             Display version
+  --log LOG_FILE           Write log output to this file, if present. If set to
+                           `[OUTPUT]`, log output is sent to the LSP client.
+  --stdio                  Use stdio (this flag is ignored, stdio is always
```

Comment:
Aha! Let's then comment that this is used as communication protocol between the client and server. You can also comment that for now we don't support any other methods, so specifying this flag doesn't do anything since it is the only and also default method.

Btw this flag for example -> it is actually a flag that should belong to the `serve` command, right? If `waspls` was organized in such way that it had top level commands under which we scope other stuff, so `version` and `serve`.